packages/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.22
+
+* Add dummy map initialization
```

Comment:
May be worth clarifying that only for Android.

Like this:
![image](https://user-images.githubusercontent.com/18647037/72849499-de087780-3cb7-11ea-8e99-9edf06075ff8.png)

Reply:
Done